doc/nrf/ug_thingy91.rst

```diff
@@ -305,3 +305,21 @@ To build and program the source code from the command line, complete the followi
        west flash
 
    #. The device will reset and run the programmed sample or application.
+
+
+Using RSA signing for MCUboot
+=============================
+
+In order to be compatible with the factory-programmed bootloader on Thingy:91, the signing algorithm must be changed from the MCUboot default.
+
+One way to achieve this, is to create the file :file:`child_image/mcuboot.conf` in your application folder and add the following content:
+
+.. parsed-literal::
+   :class: highlight
+
+   CONFIG_BOOT_SIGNATURE_TYPE_RSA=y
+   CONFIG_BOOT_SIGNATURE_KEY_FILE="root-rsa-2048.pem"
+
+These configurations are picked up by the build system and enable RSA signing of the images for backward compatibility with the MCUboot versions that precede the |NCS| v1.4.0.
```

Comment:
**suggestion**: add a note to make users aware that this is an insecure operation, and should only be done for development purposes.

```suggestion
These configurations are picked up by the build system and enable RSA signing of the images for backward compatibility with the MCUboot versions that precede the |NCS| v1.4.0.

.. note::

   This configures MCUBoot to use the default RSA keys, so that it is compatible with the factory-provisioned bootloader present on the Thingy:91.
   They should only be used for development.
   In production you should use your own keys, which must be kept secret. See :ref:`ug_fw_update_development_keys` for more information.
```

Reply:
Thanks, was looking for a good reference for that. Uma added the note and link. Please have a look if it's good now